pkg/universe.dagger.io/bash/bash.cue

```diff
@@ -13,15 +13,18 @@ import (
 
 // Like #Run, but with a pre-configured container image.
 #RunSimple: #Run & {
-	_simpleImage: #SimpleImage
+	_simpleImage: #Image
 	input:        _simpleImage.output
 }
 
-// Build a simple container image which can run bash
-#SimpleImage: alpine.#Build & {
+// Default simple container image which can run bash
+#Image: alpine.#Build & {
 	packages: bash: _
 }
 
+// DEPRECATED: Use bash.#Image instead
```

Comment:
Nice one, will it disappear on a specific on a specific version or we stay backward compatible forever?

Reply:
I also like the idea of knowing when something is scheduled for removal. `0.3.0` might make sense in this case.

Reply:
Me too, but we don't have a deprecation plan.

Reply:
> Nice one, will it disappear on a specific on a specific version or we stay backward compatible forever?

Not forever, I imagine there'll come a time when we remove deprecated code. At least most of it. Can be in `0.3.0`. But we need the warnings first, to warn users in time.